hack/verify-goimports.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+goimports=$(realpath "$1")
```

Comment:
realpath is GNU coreutils only.
Thought: Is this a problem? I imagine every open source dev installs them even if not using Linux....

Reply:
I think we already require coreutils in other parts of the codebase already so I don't think it's an issue

Reply:
> realpath is GNU coreutils only.

Useful to know, thanks - as James says we probably just have a hard dependency on GNU coreutils at least for now (verify-goimports is basically copy-pasted from hack/verify-gofmt, which uses `realpath`).

That dependency could be better documented, though - I'll maybe look at adding it somewhere :+1: